homeassistant/components/synology_dsm/common.py

```diff
@@ -62,22 +63,24 @@ def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
             self.config_url = f"http://{entry.data[CONF_HOST]}:{entry.data[CONF_PORT]}"
 
         # DSM APIs
+        self.file_station: SynoFileStation | None = None
         self.information: SynoDSMInformation | None = None
         self.network: SynoDSMNetwork | None = None
+        self.photos: SynoPhotos | None = None
         self.security: SynoCoreSecurity | None = None
         self.storage: SynoStorage | None = None
-        self.photos: SynoPhotos | None = None
         self.surveillance_station: SynoSurveillanceStation | None = None
         self.system: SynoCoreSystem | None = None
         self.upgrade: SynoCoreUpgrade | None = None
         self.utilisation: SynoCoreUtilization | None = None
 
         # Should we fetch them
         self._fetching_entities: dict[str, set[str]] = {}
+        self._with_file_station = True
         self._with_information = True
+        self._with_photos = True
```

Comment:
Looks like an unrelated change?

Reply:
yeah, i've just sorted these properties alphabetical - some has been done above for the `# DSM APIs` and below `# Determine if we should fetch an API`